src/container/containerManager.ts

```diff
@@ -143,6 +142,18 @@ export class ContainerManager {
         };
     }
 
+    private getDeployFileName(templateFileName: string, templateSchemaVersion: string): string {
+        const platform = templateSchemaVersion > "0.0.1" ? `.${Platform.getDefaultPlatform().platform}` : "";
```

Comment:
String compare may have issue: e.g. "10.0" < "2.0"

Reply:
yes. If we need version compare later, I will use utility to do so. Maybe from other existing version comparison npm. But for the current scenario, the compare works. Since the first char is "0".